src/cmd/services/m3dbnode/config/limits.go

```diff
@@ -55,14 +60,12 @@ type LimitsConfiguration struct {
 	MaxEncodersPerBlock int `yaml:"maxEncodersPerBlock" validate:"min=0"`
 }
 
-// MaxRecentlyQueriedSeriesBlocksConfiguration sets the upper limit on time
-// series blocks count within a given lookback period. Queries which are issued
-// while this max is surpassed encounter an error.
-type MaxRecentlyQueriedSeriesBlocksConfiguration struct {
-	// Value sets the max recently queried time series blocks for the given
-	// time window.
+// MaxRecentQueryResourceLimitConfiguration sets an upper limit on resources consumed by all queries
+// globally within a dbnode per some lookback period of time. Once exceeded, queries within that period
+// of time will be abandoned.
+type MaxRecentQueryResourceLimitConfiguration struct {
+	// Value sets the max value for the resource limit.
```

Comment:
nit Would this make more sense as `Limit`?

Reply:
Eh I think for backwards compat we are committed here